torchvision/models/detection/retinanet.py

```diff
@@ -455,7 +521,11 @@ def postprocess_detections(self, head_outputs, anchors, image_shapes):
 
         return detections
 
-    def forward(self, images, targets=None):
+    def forward(
+        self,
+        images: List[Tensor],
+        targets: Optional[Dict[str, Tensor]] = None,
+    ):
```

Comment:
Here as well. I'm unsure of type of the output.

Reply:
Similar to above, see here:
https://github.com/pytorch/vision/blob/3e7653c8c4b5eb0c854251a0fb86be18de55bc68/torchvision/models/detection/ssd.py#L284-L285